src/olympia/reviewers/utils.py

```diff
@@ -166,7 +170,7 @@ def render_score(self, value):
 
 
 class PendingManualApprovalQueueTable(AddonQueueTable):
-    addon_type = tables.Column(verbose_name=_('Type'), empty_values=())
+    addon_type = tables.Column(verbose_name=_('Type'), accessor='type', orderable=False)
```

Comment:
you meant to drop `empty_values` too I assume.

Reply:
Ah yep good call